R/system.R

```diff
@@ -1297,10 +1297,18 @@ submitGoogleBigQueryJob <- function(project, sqlquery, destination_table, write_
 #' @export
 getDataFromGoogleBigQueryTable <- function(project, dataset, table, page_size = 10000, max_page, tokenFileId, max_connections = 8){
   if(!requireNamespace("bigrquery")){stop("package bigrquery must be installed.")}
+  # Begin Workaround ref: https://github.com/r-dbi/bigrquery/issues/395
+  # Remember original scipen
+  original_scipen = getOption("scipen")
+  # Temporary override it with 20 to workaround: Invalid value at 'start_index' (TYPE_UINT64), "1e+05" [invalid] error
+  options(scipen = 20)
   token <- getGoogleTokenForBigQuery(tokenFileId)
   bigrquery::set_access_cred(token)
   tb <- bigrquery::bq_table(project = project, dataset = dataset, table = table)
-  bigrquery::bq_table_download(tb,  page_size = page_size, max_results = max_page, quiet = TRUE, max_connections = max_connections)
+  df <- bigrquery::bq_table_download(tb,  page_size = page_size, max_results = max_page, quiet = TRUE, max_connections = max_connections)
+  # Set original scipen
+  options(scipen = original_scipen)
```

Comment:
Make sure this reverting of scipen is done even in error case, using tryCatch.

Reply:
Changed it to use tryCatch.